src/components/NumberCard/index.jsx

```diff
@@ -133,6 +95,45 @@ class NumberCard extends React.Component {
     }
   }
 
+  addEvents() {
+    document.addEventListener('click', this.handleDocumentClick, true);
+  }
+
+  // eslint-disable-next-line react/no-unused-class-component-methods
```

Comment:
nit: is it worth adding a comment here explaining why we need to keep this component method is if not used within the class?

Reply:
I couldn't find a usage for it in the component but there is a test for it (closeDetails closes the detail actions) which fails on removal, the test could be removed along with the method.